src/entity/Car.java

```diff
@@ -0,0 +1,40 @@
+package entity;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Car {
+    private int id;
+    private AtomicBoolean free = new AtomicBoolean(true);
+    private static final Logger logger = Logger.getLogger(Car.class.getName());
+
+    public Car(int id) {
+        this.id = id;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public AtomicBoolean isFree() {
+        return free;
+    }
+
+    public void occupy(Customer customer) {
```

Comment:
Generally, method params are final, if you don't think to change their, inside method logic.